src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -190,4 +190,28 @@ private function isRequired(Constraint $constraint): bool
 
         return false;
     }
+
+    /**
+     * @param Constraint[] $propertyConstraints
+     *
+     * @return Constraint[]
+     */
+    private function flattenConstraints(array $propertyConstraints): array
+    {
+        $constraints = [];
+
+        foreach ($propertyConstraints as $propertyConstraint) {
+            if ($propertyConstraint instanceof AtLeastOneOf) {
+                continue;
```

Comment:
Why continuing in this case? I think we should take the constraints of `AtLeastOneOf` too for the property metadata, don't you think?

Reply:
Hmm.. Not sure for real, because:
```
@Assert\AtLeastOneOf({
    @Assert\Length(min=1, max=2),
    @Assert\Length(min=5, max=6)
})
```
The last one would override the first one and then the schema would be wrong :thinking: